src/invidious/routes/subscriptions.cr

```diff
@@ -106,31 +106,7 @@ module Invidious::Routes::Subscriptions
         env.response.headers["content-disposition"] = "attachment"
         playlists = Invidious::Database::Playlists.select_like_iv(user.email)
 
-        return JSON.build do |json|
-          json.object do
-            json.field "subscriptions", user.subscriptions
-            json.field "watch_history", user.watched
-            json.field "preferences", user.preferences
-            json.field "playlists" do
-              json.array do
-                playlists.each do |playlist|
-                  json.object do
-                    json.field "title", playlist.title
-                    json.field "description", html_to_content(playlist.description_html)
-                    json.field "privacy", playlist.privacy.to_s
-                    json.field "videos" do
-                      json.array do
-                        Invidious::Database::PlaylistVideos.select_ids(playlist.id, playlist.index, limit: 500).each do |video_id|
-                          json.string video_id
-                        end
-                      end
-                    end
-                  end
-                end
-              end
-            end
-          end
-        end
+        return Invidious::User::Export.to_invidious(user)
```

Comment:
You can also remove the unnecessary DB call right above this line (I can't review outside of the code you changed) ^^

Reply:
Good catch!